xml/System.Web.Script.Serialization/JavaScriptSerializer.xml

```diff
@@ -16,14 +16,19 @@
   <Interfaces />
   <Docs>
     <summary>
-      <see href="https://www.newtonsoft.com/json">Json.NET</see> should be used for serialization and deserialization. Provides serialization and deserialization functionality for AJAX-enabled applications.</summary>
+      Provides serialization and deserialization functionality for AJAX-enabled applications.
```

Comment:
If we are truly recommending against this in favor of  `Newontosft.Json` for < netfx 4.7.2, should we consider obsoleting the API in .NET Core going forward or is there no need for that/the usage too high? Of course for net4.7.2+ and .net core `S.T.Json` is the way to go :)

cc @terrajobst, @GrabYourPitchforks